src/Monolog/Formatter/JsonFormatter.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ErrorControl\Monolog\Formatter;
+
+use Monolog\Formatter\JsonFormatter as MonologJsonFormatter;
+use Throwable;
+
+class JsonFormatter extends MonologJsonFormatter
+{
+    public function __construct(
+        int $batchMode = MonologJsonFormatter::BATCH_MODE_JSON,
+        bool $appendNewline = true,
+        bool $ignoreEmptyContextAndExtra = false,
+        bool $includeStacktraces = true,
+    ) {
+        parent::__construct(
+            $batchMode,
+            $appendNewline,
+            $ignoreEmptyContextAndExtra,
+            $includeStacktraces,
+        );
+    }
+
+    protected function normalizeException(Throwable $e, int $depth = 0): array
+    {
+        $data = parent::normalizeException($e, $depth);
+
+        if ($this->includeStacktraces) {
+            $data['trace'] = $e->getTraceAsString();
+        }
```

Comment:
Super.. kdyz jsme to popisoval, myslel jsem, ze bude potreba tu `normalizeException` preimplmentovat, ale tohle je mnohem lepsi :+1: